### ISSUE TITLE
Add Fluid mechanics formula

### ISSUE THREAD

**Issue by Amit366** [Amit366/FunwithPhysics#94]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
Add different fluid mechanics formula

**Comment by Aniket-508** [comment]
Hey @Amit366 , can I take up this issue?

**Comment by Amit366** [comment]
@Aniket-508 u can work on it for any doubts join the discord channel whose link is in readme

**Comment by Amit366** [comment]
@Aniket-508 there are many formulas so separate it just like in work power energy calculator

**Comment by Aniket-508** [comment]
![fluid](https://user-images.githubusercontent.com/71957423/135754494-f2623e20-2856-4ed1-b061-c366bf8c06c4.JPG)
Pls check if all these equations are correct and suggest some if not required or need to be added.

**Comment by Amit366** [comment]
@Aniket-508 yes these are correct and add it separately. Add ask these doubts on discord as it will be easy there
